test/content-get.js

```diff
@@ -0,0 +1,45 @@
+const request = require("request");
+const {promisify} = require('util')
+const fs = require('fs')
+const writeFile = promisify(fs.appendFile)
```

Comment:
rename to appendFile because this is misleading, also `fs` offers sync libraries like `writeFileSync` etc.